proto/raft_cmdpb.proto

```diff
@@ -220,7 +220,8 @@ message RollbackMergeRequest {
 message RollbackMergeResponse {}
 
 message SwitchWitnessRequest {
-    metapb.Peer peer = 1;
+    uint64 id = 1;
```

Comment:
Should it be repeated?

Reply:
I think should be. eg: distributed across different stores

Reply:
Fixed, thanks!